contents/docs/developing-locally.mdx

```diff
@@ -27,6 +27,8 @@ git clone https://github.com/PostHog/posthog
 docker-compose -f docker-compose.dev.yml up -d redis db
```

Comment:
I think we need to update this line to:

```
docker-compose -f docker-compose.dev.yml up -d
./bin/start-frontend
```

Reply:
I'm not sure if we want to as for backend developers its better to run the backend locally